ReactWindows/ReactNative/Views/DatePicker/ReactDateTimePickerManager.cs

```diff
@@ -0,0 +1,213 @@
+        {
+            if (date.HasValue)
+            {
+                datePicker.Date = date.Value;
+                timePicker.Time = date.Value.TimeOfDay;
+            }
+        }
+
+        /// <summary>
+        /// Sets the maximum allowed year of the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="date">The value to set as maximum.</param>
+        [ReactProp("maxYear")]
+        public void SetMaxYear(StackPanel view, DateTime? date)
+        {
+            if (date.HasValue)
+            {
+                datePicker.MaxYear = date.Value;
+            }
+        }
+
+        /// <summary>
+        /// Sets the minimum allowed year of the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="date">The value to set as minimum.</param>
+        [ReactProp("minYear")]
+        public void SetMinYear(StackPanel view, DateTime? date)
+        {
+            if (date.HasValue)
+            {
+                datePicker.MinYear = date.Value;
+            }
+        }
+
+        /// <summary>
+        /// This method returns the <see cref="ReactDatePickerShadowNode"/>
+        /// which will be then used for measuring the position and size of the
+        /// view.
+        /// </summary>
+        /// <returns>The shadow node instance.</returns>
+        public override ReactDateTimePickerShadowNode CreateShadowNodeInstance()
+        {
+            return new ReactDateTimePickerShadowNode();
+        }
+
+        /// <summary>
+        /// Implement this method to receive optional extra data enqueued from
+        /// the corresponding instance of <see cref="ReactShadowNode"/> in
+        /// <see cref="ReactShadowNode.OnCollectExtraUpdates"/>.
+        /// </summary>
+        /// <param name="root">The root view.</param>
+        /// <param name="extraData">The extra data.</param>
+        public override void UpdateExtraData(StackPanel root, object extraData)
+        {
+        }
+
+        /// <summary>
+        /// Unbind event handlers
+        /// </summary>
+        /// <param name="reactContext">The React context.</param>
+        /// <param name="view">The view.</param>
+        public override void OnDropViewInstance(ThemedReactContext reactContext, StackPanel view)
+        {
+            base.OnDropViewInstance(reactContext, view);
+            datePicker.DateChanged -= OnDateChanged;
+            timePicker.TimeChanged -= OnTimeChanged;
+        }
+
+        /// <summary>
+        /// Creates a new view instance of type <see cref="StackPanel"/>.
+        /// </summary>
+        /// <param name="reactContext">The React context.</param>
+        /// <returns>The view instance.</returns>
+        protected override StackPanel CreateViewInstance(ThemedReactContext reactContext)
+        {
+            panel = new StackPanel();
+            panel.Orientation = Orientation.Vertical;
```

Comment:
I'd rather not have a composite view like this in core, this is just as easy to build at the JavaScript level...